ivadomed/main.py

```diff
@@ -362,185 +415,401 @@ def run_command(context, n_gif=0, thr_increment=None, resume_training=False, no_
             * If "segment" command: No return value.
 
     """
-    command = copy.deepcopy(context[ConfigKW.COMMAND])
+    # Command can only be: segment/train/test
+    command: str = copy.deepcopy(context[ConfigKW.COMMAND])
+
+    #################################
+    #  Loader Version Determination:
+    #################################
+    # If the "contrast_params" key is present, we utilize that new loader pathway.
```

Comment:
```suggestion
    # If the "contrast_params" key is not present, we utilize that new loader pathway.
```

IIUC the code below:
"contrast_param" present = TRADITIONAL_BIDS_LOADER
"contrast_param" not present = MULTI_PATH_LOADER (V2 loader)

Reply:
This direction is absolutely amazing. I was afraid of breaking stuff in our current pipeline until everything with multi-contrasts & multi-session lands in place but just dividing till the multi-contrast & multi-session pipeline would give us time to fix if there are any hiccups that are realized later on.

Reply:
Good point. This is the relevant CLI section you were talking about. I will play around with the trigger to use simpler conditions: 

![image](https://user-images.githubusercontent.com/6720068/201740844-82aa4e12-fb8d-45ff-b862-0cf004623440.png)

Reply:
Now updated to check solely for the three new keys as per suggested. Worked like a charm.